linux/Instructions_for_destkop_app

```diff
@@ -0,0 +1,3 @@
+edit the Bluebubbles.desktop change exec to the location of the bluebubbles_app
+edit the icon to the location of the icon usually in flutter_assets
+move Bluebubbles.desktop to /usr/share/applications(or where ever your applications folder is)
```

Comment:
Might be nice to change this into a markdown file or something rather than a file with no association. 

Also minor nitpick but desktop is spelled wrong in the filename :)